src/controller/Controller.java

```diff
@@ -1,29 +0,0 @@
-package controller;
-import model.LabyrinthMatrix;
-import view.LabyrinthText;
-
-public class Controller {
-	public static void main(String[] args) {
-		LabyrinthMatrix l = new LabyrinthMatrix();
-		l.setupLabyrinth(-1, 2, 0, 1);
-		
-		l.loadFromFile("input.txt");
-		
-		LabyrinthText lt = new LabyrinthText();
-		lt.setLabyrinth(l);
```

Comment:
[info]
Same as above, why not merge the setter with the constructor? The LabText has no meaning without the lab itself. And if you may one day wish for the text to represent another lab, you could leave the setter there, but only to that extent.